src/main/java/com/seleniumeasy/listbox/JQueryListBoxPage.java

```diff
@@ -0,0 +1,114 @@
+package com.seleniumeasy.listbox;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import com.seleniumeasy.DemoHomePage;
+
+import static com.codeborne.selenide.CollectionCondition.size;
+import static com.codeborne.selenide.Condition.exactText;
+import static com.codeborne.selenide.Condition.exist;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+
+public class JQueryListBoxPage extends DemoHomePage {
+    private final SelenideElement buttonAdd = $(".pAdd"),
+            buttonAddAll = $(".pAddAll"),
+            buttonRemove = $(".pRemove"),
+            buttonRemoveAll = $(".pRemoveAll");
+    private final ElementsCollection pickList = $$(".pickData option"),
+            resultList = $$(".pickListResult option");
+
+    public JQueryListBoxPage addItemToResultList(String name) {
+        pickList.find(exactText(name)).click();
+        buttonAdd.click();
+        return this;
+    }
+
+    public JQueryListBoxPage addItemsToResultList(String[] names) {
+        for (String name : names) {
+            pickList.find(exactText(name)).click();
+        }
+        buttonAdd.click();
+        return this;
+    }
+
+    public JQueryListBoxPage removeItemFromResultList(String name) {
```

Comment:
method duplication: 
please merge removeItemFromResultList(String name) and removeItemsFromResultList(String[] names)
you can use VarArgs

Reply:
Delete all duplicate methods in JQueryListBoxPage class. Replace parameters by VarArgs. Also, modify tests